model/parameter.go

```diff
@@ -752,23 +752,15 @@ func (p ParameterDefinition) validateFileLogSortType(_ parameterFieldType, value
 func (p ParameterDefinition) validateFileLogSortTypeFields(value map[string]any) error {
 	for s, n := range value {
 		switch strings.ToLower(s) {
-		case "layout", "location", "sorttype", "regexkey":
+		case "layout", "location", "sorttype", "regexkey", "sortdirection":
 			_, ok := n.(string)
 			if !ok {
 				return stanzaerrors.NewError("incorrect type included in 'sort_rules' field",
 					"filelog"+s+"should be of type string")
 			}
-
-		case "ascending":
-			_, ok := n.(bool)
-			if !ok {
-				return stanzaerrors.NewError("incorrect type included in "+s+" field",
-					"filelog"+s+"should be of type bool")
-			}
 		default:
 			return stanzaerrors.NewError("unexpected field "+s+" included in struct",
 				s+"should not be an included field in filelogsort")
```

Comment:
I know this isn't part of this PR, but shouldn't there be a space here?
```suggestion
				s+" should not be an included field in filelogsort")
```

Reply:
Updated.